lib/tasks/bower-install.js

```diff
@@ -12,13 +12,20 @@ module.exports = Task.extend({
   },
   // Options: Boolean verbose
   run: function(options) {
+    var existsSync     = require('exists-sync');
     var chalk          = require('chalk');
     var bower          = this.bower;
+    var bowerJson      = path.join(this.project.root, 'bower.json');
```

Comment:
`this.project.root + '/bower.json'` is sufficient, path isn't needed as all node `fs` operations already normalize.